index.html

```diff
@@ -30,16 +39,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box-1">Box 1</div>
+                <div class="box-2">Box 2</div>
+                <div class="box-3">Box 3</div>
+                <div class="box-4">Box 4</div>
+                <div class="box-5">Box 5</div>
+                <div class="box-6">Box 6</div>
+                <div class="box-7">Box 7</div>
+                <div class="box-8">Box 8</div>
+                <div class="box-9">Box 9</div>
+                <div class="box-10">Box 10</div>
```

Comment:
so you could've done this with 2 classes so that you would be able to edit all of them and each one individually class = 'box box-1'